utils.go

```diff
@@ -53,6 +53,14 @@ func IsDocumentation(path string) bool {
 	return data.DocumentationMatchers.Match(path)
 }
 
+func GetMimeType(language string) string {
+	if mime, ok := data.LanguagesMime[language]; ok {
```

Comment:
If you keep languages with a string zero value assigned in the map, this code will always return `ok` set to true